pkg/provider/keepersecurity/client_test.go

```diff
@@ -551,33 +533,11 @@ func TestClientPushSecret(t *testing.T) {
 			},
 			wantErr: false,
 		},
-		{
```

Comment:
Nice, but please add tests for the new finder logic. Basically, the case where it returns more than one secret with the same title.